packages/components/app/styles/components/alert.scss

```diff
@@ -0,0 +1,263 @@
+//
+// ALERT
+//
+// properties within each class are sorted alphabetically
+//
+//
+
+@use "../mixins/focus-ring" as *;
+
+.hds-alert {
+  align-items: flex-start;
+  display: flex;
+}
+
+// ICON
+
+.hds-alert__icon {
+  flex: none;
+  height: 20px;
+  margin-right: 12px;
+  width: 20px;
+
+  .hds-alert--type-compact & {
+    height: 14px;
+    width: 14px;
+  }
+}
+
+// CONTENT (TEXT + ACTIONS + GENERIC)
+
+.hds-alert__content {
+  flex: 1 1 auto;
+}
+
+// TEXT (TITLE & DESCRIPTION)
+
+.hds-alert__text {
+  color: var(--token-color-foreground-warning-on-surface);
+  display: flex;
+  flex-direction: column;
+  font-family: var(--token-typography-body-200-font-family);
+  font-size: var(--token-typography-body-200-font-size);
+  justify-content: center;
+  line-height: var(--token-typography-body-200-line-height);
+
+  .hds-alert--type-compact & {
+    font-family: var(--token-typography-body-100-font-family);
+    font-size: var(--token-typography-body-100-font-size);
+    line-height: var(--token-typography-body-100-line-height);
+  }
+}
+
+.hds-alert__title {
```

Comment:
why do you think so? It makes sense (I have used `hds-alert__text` as place where to define the text style, because both `title` and `description` share the same font-family/font-size/line-height). introducing custom styles for everything goes against the purpose of having base/core styles (unless there are real reasons/differences, which is not the case here).

Reply:
I'm closing this discussion after our meeting. We'll keep it as is.